src/coreclr/src/jit/emitxarch.cpp

```diff
@@ -14282,6 +14295,26 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             result.insLatency    = PERFSCORE_LATENCY_1C;
             break;
 
+        case INS_movd:
+            if (memAccessKind == PERFSCORE_MEMORY_NONE)
+            {
+                // movd   r32, xmm   or  xmm, r32
+                result.insThroughput = PERFSCORE_THROUGHPUT_1C;
+                result.insLatency    = PERFSCORE_LATENCY_3C;
```

Comment:
As I read the document, this should be `PERFSCORE_LATENCY_2C`

Reply:
OK, I can change this,